src/clib/lib.rs

```diff
@@ -4,7 +4,7 @@ use serde_json;
 use std::ffi::CString;
 
 #[no_mangle]
-pub extern "C" fn nispor_state_get(
+pub extern "C" fn nispor_retrieve(
```

Comment:
How about keep the old name or use `nispor_net_state_retrieve` which better align with rust API.